shadows/framework/src/main/java/org/robolectric/shadows/ShadowAppOpsManager.java

```diff
@@ -394,17 +396,17 @@ protected int noteProxyOpNoThrow(
     return checkOpNoThrow(op, proxiedUid, proxiedPackageName);
   }
 
+  @RequiresApi(api = Build.VERSION_CODES.S)
   @Implementation(minSdk = Build.VERSION_CODES.S)
   protected int noteProxyOpNoThrow(
```

Comment:
To do this it will need `looseSignatures = true`  in the `@Implements(...)` annotation on top, and all of the args of this method need to be converted to `Object`.

Reply:
It's first time to know `looseSignatures`. I will give a try.

Reply:
Thanks. The `llooseSignatures` works for me.